### ISSUE TITLE
Registry Mirror configuration not recognized

### ISSUE THREAD

**Issue by thecloudgarage** [aws/eks-anywhere#5962]
**What happened**:
Proxy cache configuration is not working as per documentation. OCI Namespace is not a recognized field. Cluster configuration snip is provided below. Cluster is successfully created if registry configuration is omitted. 
![image](https://github.com/aws/eks-anywhere/assets/39495790/7ddf382b-1b1d-4436-9b8d-2bbe9e8cc47c)
I have also exported the certificate of the Harbor registry and I can validate the SSL connection
```
export EKSA_REGISTRY_MIRROR_CA="/home/ubuntu/common-certs/tls.crt"
```
After attempting to create the cluster...I am getting the error
```
Error: the cluster config file provided is invalid: unable to parse /home/ubuntu/test119-eks-a-cluster.yaml file: error unmarshaling JSON: while decoding JSON: json: unknown field "ociNamespaces"
````
**Environment**: vSphere, Harbor configured with a proxy-cache project named proxy.docker.io and a registry endpoint with docker hub (validated)
- EKS Anywhere Release: v0.12.2
- EKS Distro Release: eksdRelease:kubernetes-1-23-eks-8

**Comment by ivyostosh** [comment]
@thecloudgarage the registry mirror configuration feature is available in eks anywhere v0.14+. Please upgrade your eks anywhere version (to the latest if possible) and retry.

**Comment by thecloudgarage** [comment]
Thanks... working
